src/features/app/Head.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import Helmet from 'react-helmet'
+
+interface HeadProps {
+  data?: {
+    title?: string
+    pathname?: string
+    description?: string
+    image?: string
+    imageAlt?: string
+  }
+}
+
+const Head: React.FunctionComponent<HeadProps> = ({ data = {}, children }) => {
+  const {
+    title,
+    pathname,
+    description,
+    image,
+    imageAlt
+  } = data
+
+  return (
+    <>
+      <Helmet>
+        <meta name="twitter:card" content="summary" />
+        <meta name="twitter:site" content="@qri_io" />
+        <meta property="og:type" content="website" />
+        <meta property="og:site_name" content="Qri Cloud"/>
+
+        { title && <title>{title}</title> }
```

Comment:
Can we use 1 && operator and wrap the content with React.Fragment aka <></> ?

Reply:
Instead of this : 
`
{ title && <title>{title}</title> }
{ title && <meta name="twitter:title" content={title} /> }
{ title && <meta property="og:title" content={title} /> }
`
lets try something like this:
`
{ title && 
<>
  <title>{title}</title>
  <meta name="twitter:title" content={title} />
  <meta property="og:title" content={title} />
</>}
`